c2corg_ui/static/js/map/map.js

```diff
@@ -114,9 +143,19 @@ app.MapController = function($scope, mapFeatureCollection) {
       new ol.layer.Tile({
         source: new ol.source.OSM()
       })
-    ]
+    ],
+    view: new ol.View({
+      center: ol.extent.getCenter(app.MapController.DEFAULT_EXTENT),
+      zoom: app.MapController.DEFAULT_ZOOM
+    })
   });
 
+  /**
+   * @type {ol.View}
+   * @private
+   */
+  this.view_ = this.map.getView();
+
   // editing mode
   if (this['edit']) {
```

Comment:
Not your code, but the edit variable should not be accessed using the bracket notation but declared and accessed using the point notation. In addition, the constructor should be marked `@struct` to avoid accessing inexistant variables.